deploy-deps.sh

```diff
@@ -131,6 +132,15 @@ deploy_smee() {
     kubectl apply -k "${script_path}/dependencies/smee"
 }
 
+deploy_kyverno() {
+    kubectl apply -k "${script_path}/dependencies/kyverno" --server-side
+    # retry "kubectl wait --for=condition=Ready --timeout=120s -l app.kubernetes.io/instance=kyverno -n kyverno pod" \
+    #       "Kyverno did not become available within the allocated time"
+    # Wait for policy CRD to be installed. Don't need to wait for everything to be up
+    sleep 5
```

Comment:
Why not use the retry?

Reply:
Had to figure out what exactly to retry on. I didn't want to have to wait for things to be up, just wanted the CRD to be there so CRs can be created. Maybe retrying on getting the CRD would have worked.